packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -115,6 +116,14 @@ export default function SquadHeaderMenu({
       });
     }
 
+    list.push({
+      icon: <ContextMenuIcon Icon={FlagIcon} />,
+      action: () => {
+        openModal({ type: LazyModal.ReportSquad });
```

Comment:
I would think this needs to pass some props right?

Reply:
We should not need this whole block anymore, on line 113, this was introduced.